source/features/useful-not-found-page.tsx

```diff
@@ -71,30 +71,30 @@ async function getUrlToFileOnDefaultBranch(): Promise<string | void> {
 }
 
 async function showMissingPart(): Promise<void> {
-	const parts = parseCurrentURL();
-	const bar = <h2 className="container mt-4 text-center"/>;
-
-	for (const [i, part] of parts.entries()) {
-		if (i === 2 && ['tree', 'blob', 'edit'].includes(part)) {
-			// Exclude parts that don't exist as standalones
-			continue;
-		}
-
-		if (i === parts.length - 1) {
-			// The last part of the URL is a known 404
-			bar.append(' / ', getStrikeThrough(part));
-		} else {
-			const pathname = '/' + parts.slice(0, i + 1).join('/');
-			bar.append(i ? ' / ' : '', <a href={pathname}>{part}</a>);
-		}
-	}
+	const pathParts = parseCurrentURL();
+	const breadcrumbs = [...pathParts.entries()]
+		.reverse() // Checks the anchors right to left
+		.map(([i, part]) => {
+			if (
+				// Exclude parts that don't exist as standalones
+				(i === 0 && part === 'orgs') // #5483
+				|| (i === 2 && ['tree', 'blob', 'edit'].includes(part))
+				|| i === pathParts.length - 1 // The last part is a known 404
```

Comment:
All the excluded parts are still shown, but are immediately crossed and not checked for 404s. Do you know of any more parts that need to be excluded?

Reply:
`blame`, `raw`? I think this part is useless most of the time, except for `commits`.